src/autogluon/bench/cloud/aws/run_deploy.py

```diff
@@ -11,6 +11,17 @@
 
 
 def construct_context(custom_configs: dict = {}):
```

Comment:
I noticed this while reading the doc strings. We probably shouldn't have this default to `{}`. We should have it be `Optional[dict] = None` and then add
```
custom_configs = {} if custom_configs is None else custom_configs
```
Default mutable function args have unexpected behavior in Python.

Reply:
Good point. Updated.